ts/sagas/startup/loadServiceRequestHandler.ts

```diff
@@ -75,7 +75,7 @@ export function* loadServiceRequestHandler(
     } else {
       throw Error();
     }
-  } catch {
+  } catch (error) {
     yield put(loadService.failure(action.payload));
```

Comment:
```suggestion
    const failureDescription = `${error.message || ""} ${action.payload}`;
    yield put(loadService.failure(failureDescription));
```
you should use error